public/i18n/en.json

```diff
@@ -245,14 +245,17 @@
   "ftm-faq-makerspace-partners": "Who are #findthemasks Makers partners?",
   "ftm-faq-makerspace-partners-answer-1": "Our <a href=\"https://findthemakers.com\">#findthemakers</a> is in partnership with <a href=\"https://www.nationofmakers.us/\">Nation of Makers</a>. Nation of Makers is a national nonprofit supporting America’s maker organizations through community building, resource sharing, and advocacy, within the maker movement and beyond. Nation of Makers is supporting maker organizations during the COVID-19 crisis by connecting them to facilities that need PPE, identifying suppliers (and discounts) for maker groups fabricating PPE, and identifying funding sources for groups producing PPE. They are also collating <a href=\"https://www.nationofmakers.us/covid-19-maker-response\"resources for makers responding to COVID-19</a>.",
   "ftm-faq-makerspace-partners-answer-2": "#findthemasks works with <a href=\"https://shield-net.org/\">Shield Net</a>, a group that matches organizations in need of medical face shields with manufacturers that produce them. They have amassed a wide network of university makerspaces and commercial manufacturers who are making face shields based on their <a href=\"https://making.engr.wisc.edu/shield/\">open-source design</a>.",
   "ftm-faq-independent-makerspace": "I'm an independent maker, how do I join a Maker Group or network?",
-  "ftm-faq-independent-makerspace-answer-1": "If you are a makerspace of any size, join their face shield network <a href=\"https://docs.google.com/forms/d/e/1FAIpQLSei2ti1aSj4e6Ky4TcPaLFfp8aa2n8TOHItIFKsD-DritAZeg/viewform\">here</a>.",
-  "ftm-faq-independent-makerspace-answer-2": "If you would like to purchase face shields at-cost, fill out their request form <a href=\"https://docs.google.com/forms/d/e/1FAIpQLSeDHmXGMf4VGZApVmMNd733DyaekIcWVMasadRvDGovMOdNMA/viewform\">here</a>.",
-  "ftm-faq-independent-makerspace-answer-3": "Shield Net only supplies face shields; for other PPE needs, fill out our #findthemasks <a class=\"add-donation-site-form\">PPE request form</a>.",
+  "ftm-faq-independent-makerspace-answer-1": "If you are an individual marker, we encourage you to use the #FindtheMakers map to connect with a makerspace or maker group in your community. We encourage this collaborative interaction to avoid pinning your home and protect your privacy.",
+  "ftm-faq-independent-makerspace-answer-2": "If you are a makerspace or maker group of any size, join the #FindtheMakers map <a href=\"https://airtable.com/shruH5B27UP3PqKgg\">here</a>.",
   "ftm-faq-independent-makerspace-answer-4": "<strong>CA Only</strong>: Find the Masks Canada also collaborates with #thePPEDrive’s call for <a href=\"https://theppedrive.com/3d-printed-visor\">3D printers</a>. Join their network to donate 3D-printed face shields.",
   "ftm-faq-sites-non-medical-grade": "Why don't all sites allow non-medical grade items?",
```

Comment:
This was probably not clear enough on the Google Doc but Stef & I used the "xx only" parts in the doc to flag content meant for country specific FAQ content bound for view/partials. This definitely works for now and up to you whether you want to include on this PR or leave it for another issue? Not sure how crazy it is with the new hiding/moving logic - so cool!! Thanks for doing this

Reply:
I thought about that, but then saw that the text was an exact match for what was already there so figured it wasn't supposed to change, hahah. I will see what I can do.

Reply:
@linderr updated so CA Only text only displays on CA